consensus/istanbul/core/core.go

```diff
@@ -37,6 +37,41 @@ import (
 	"github.com/syndtr/goleveldb/leveldb"
 )
 
+type core struct {
```

Comment:
The addition of `CurrentRoundState()`. 

`CurrentView()` is not an addition, but it lived in `handler.go` before, I've just moved it there. I removed `Sequence()` since it seemed redundant...